### PR TITLE
Closes #979 and implements schedulerName for elements that use persistent storage

### LINKED ISSUE #979: [feature request] support for configuring a different kubernetes scheduler
### Overview
Some storage solutions such as Portworx implement their own scheduler to make sure pods and volumes are colocated. A schedulerName value should be present in the values.yaml to allow this configuration in each of Harbor that uses persistent storage.

### PR DESCRIPTION
**Pull request by Jramirezg**
Closes #979 and implements schedulerName for:

trivy
core
registry
redis
jobservice
chartmuseum
database
exporter

### CODE REVIEW

**Comment by ninjadq** [comment]
hi is this PR slove the same issue with #980 ?

**Comment by Jramirezg** [comment]
@ninjadq yes it does. I put the other pull request and got a comment mentioning that the parameter should be set globally, not per deployment. Closed the #980 now

File: README.md [hunk 1]
```diff
@@ -187,6 +187,7 @@ The following table lists the configurable parameters of the Harbor chart and th
...
+| `schedulerName`                                                               | Set Scheduler name for this chart. Useful when using solutions like Portworx with Harbor                                                                                                                                                                                                                                                                                                                                                                                                                                                        | `default-scheduler`                                                        |
```
Comment (reasonerjt) [inline README.md:190]: The default value is empty string?

**Comment by reasonerjt** [comment]
@Jramirezg 
We are wrapping up for v1.8.0 by **mid October,** to make the PR merged could you please:
1. Sign the DCO
2. Resolve conflict, squash the commit
3. Clarify the reason for introducing this parameter in the commit message.

**Comment by reasonerjt** [comment]
due to lack of response, moving this PR out of the scope of v1.8.0

**Pull request closed without merging**